pkg/command/plugin.go

```diff
@@ -458,6 +463,7 @@ func newSyncPluginCmd() *cobra.Command {
 	var syncCmd = &cobra.Command{
 		Use:   "sync",
 		Short: "Sync the plugins",
+		Long:  `We no longer sync standalone plugins. With a centralized approach to discovering plugins, synchronizing standalone plugins would install ALL plugins available for ALL products.Instead, we only synchronize any plugins that are specifically specified by the contexts. Note: to install all plugins for a specific product, plugin groups will need to be used`,
```

Comment:
```suggestion
		Long:  `Installs all recommended plugins from the active contexts. Plugins installed with this command will only be available until the context for which this plugins is active and might not be visible when the context is not active`,
```

cc @vuil @marckhouzam can you please provide your feedback as well to see if this looks fine?